### ISSUE TITLE
Can't bind to 'inputs' since it isn't a known property of 'ngx-carousel'.

### ISSUE THREAD

**Issue by Mdhaker** [sheikalthaf/ngx-carousel#61]
Hi, I'm running angular 5.0.3 and installed ngx-carousel  `"ngx-carousel": "^1.3.5"` 
I'm trying the code in the demo trail!
```
<ngx-carousel [inputs]="carouselTile" (carouselLoad)="carouselTileLoad($event)">

    <ngx-tile NgxCarouselItem *ngFor="let Tile of carouselTileItems">
        <h1>{{Tile + 1}}</h1>
    </ngx-tile>

    <button NgxCarouselPrev class='leftRs'>&lt;</button>
    <button NgxCarouselNext class='rightRs'>&gt;</button>
</ngx-carousel>
```
and I have a template parse error in the console saying: 

> Can't bind to 'inputs' since it isn't a known property of 'ngx-carousel'.

**Comment by sheikalthaf** [comment]
@Mdhaker try this carousel with universal support in another repo [@ngu/carousel](https://www.npmjs.com/package/@ngu/carousel).

**Comment by Mdhaker** [comment]
@sheikalthaf I already did and always the same error

**Comment by sheikalthaf** [comment]
@Mdhaker I tried with `Angular 5.0.5` but haven't faced the issue

**Comment by nidhi1612** [comment]
Hi,
we wanted to use ngz-carousel with our ionic app. Below are version specifications, we are also facing the same issue. Can you guide us with the solution. Is it possible to use ngx-carousel with ionic 3 application?

"@angular/common": "4.1.3",
    "@angular/compiler": "4.1.3",
    "@angular/compiler-cli": "4.1.3",
    "@angular/core": "4.1.3",
    "@angular/forms": "4.1.3",
    "@angular/http": "4.1.3",
    "@angular/platform-browser": "4.1.3",
    "@angular/platform-browser-dynamic": "4.1.3",
     "ionic-angular": "3.5.3",

**Comment by sheikalthaf** [comment]
@nidhi1612 yes it supported. But I need some time to test. I recommend you to use [@ngu/carousel](https://github.com/sheikalthaf/ngu-carousel) because your angular version is 4+

**Comment by nidhi1612** [comment]
Hi @sheikalthaf  I am still facing the same issue when i use @ngu?carousel. :( can you please tell me how to proceed further.

**Comment by heena22626** [comment]
@mdhaker I'm also facing the same issue, if you resolved it please let me know.

**Comment by RazorV** [comment]
Just check did you import NgxCarouselModule in module with your component, its not always App module.

**Comment by JigneshPatel04** [comment]
It is working in my app. 
details.moduls.ts : - import { NgxCarouselModule } from 'ngx-carousel';
@NgModule({
  imports: [
   ------,
   ------
    NgxCarouselModule
  ],
})

follow this other :https://www.npmjs.com/package/ngx-carousel

**Comment by thovo** [comment]
I just scratch my head off with this issue, I already tried to add this in `app.module.ts` and next is my sub-module but it didn't work either. Where should I import it or did I miss something?

**Comment by sajidali2444** [comment]
same problem with 5.2.1 angular version any fix or any solution, please.
can someone me in solving the problem.
Can't bind to 'inputs' since it isn't a known property of 'ngx-carousel'

**Comment by sheikalthaf** [comment]
solution for this issue is in #82 . @Mdhaker @nidhi1612 @heena22626 @RazorV
